internal/lib/utils.go

```diff
@@ -200,3 +201,26 @@ func Difference(a, b []string) []string {
 	}
 	return diff
 }
+
+func CheckConstraintsForRFC(name string, pattern string, maxlength int) bool {
+	if len(name) > maxlength {
+		utils.AviLog.Warnf("Given string is longer than expected. Maximum allowed length is: %d", maxlength)
```

Comment:
do we need to return a false here?

Reply:
@metalcyanide : You can change log statement as "String %s has length more than maximum allowed length %d"